lib/WebpackConfig.js

```diff
@@ -137,6 +137,7 @@ class WebpackConfig {
         this.vueOptions = {
             useJsx: false,
             version: null,
+            runtimeCompilerBuild: true
```

Comment:
What about making this option `boolean|undefined`, and showing the warning only if undefined ? That would allow people to configure explicitly that they need the full build, getting rid of the warning.